### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards> ActivityUpdate.json]: Voice access users are not able to to access the ‘ActivityUpdate.json’ control by giving command ‘Move to or Click ActivityUpdate.Json’.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7515]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25126.1000)
Application: Adaptive Cards UWP
Application Version: 3.0.2206.2001
Tool: Voice access

**Repro Steps:**
1. Turn on Voice access.
2. Open Adaptive Cards UWP application.
3. Command: Move to ActivityUpdate.json
4. Observe the Issue.

**Actual Result:**
Voice access users are not able to access the ‘ActivityUpdate.json’ control by giving command ‘Move to or Click ActivityUpdate.Json’. While control is accessible through ‘Show number’ commands.  
As Voice access is taking space between activity and update due to which user is not able to access the control.

**Expected Result:**
Voice access users should be able to to access the ‘ActivityUpdate.json’ control by giving command ‘Move to or Click ActivityUpdate.Json’.

**Notes:**
Issue repro with all the control which contains two words together.

**Observation:**
By giving command ‘Click Agenda’ Agenda.json is getting activated.

**User Impact:**
Voice access user will have difficulty in interacting with the controls if by visual name user is not able to activate the control and by giving command show numbers that would take much time to interact with the controls.

**MAS reference link:**
[[No Disruption of Accessibility Features - Liquid (microsoft.com)]](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01)

### Screenshots

https://user-images.githubusercontent.com/104483005/173384742-13eea87e-79ee-4a8b-863f-2f450eb3086d.mp4

<img width="960" alt="MAS4 3 1 Voice access user is not able to activate the control of activityupdate json" src="https://user-images.githubusercontent.com/104483005/173385018-c5414cfb-36f4-4e0b-8d27-58aee249aee1.png">

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vaishali1397!

I'm working on this now, and I noticed that Voice Access behaves as expected if you say "Click Activity Update" instead of "Click ActivityUpdate.json".

Is this acceptable behavior?

Thanks!

**Comment by anna-dingler** [comment]
Hi @vaishali1397,

In the Voice Access documentation, it is recommended to use the partial name when special characters are included (https://support.microsoft.com/en-us/topic/use-voice-to-interact-with-items-on-the-screen-e1b53c8a-7765-495a-9611-d9b37c008319).

I am closing this issue since Voice Access works as expected when `.json` is omitted from the command.

Please let me know if you have any questions or concerns with this resolution. Thanks!

**Comment by vaishali1397** [comment]
Hi @anna-dingler As we can observed in the video we are not able to access the 'ActivityUpdate.JSON' with Move to or click command of voice access by giving '**Move to Activity Update**' command as it contained  two joint names as **ActivityUpdate** if it named as **Activity Update.JSON** then it can be accessible.
While 'Agenda.JSON' is accessible as it contained only one word.
Uploading Issue is repro'ing...mp4…

**Comment by anna-dingler** [comment]
Hi @vaishali1397,

I am still able to bypass this issue using "Move to Activity Update" even though the name is not "Activity Update". See the gif below where I am using VoiceAccess commands.

![voiceAccess](https://user-images.githubusercontent.com/98650930/214982757-a2554b79-e4aa-4c63-a063-c0444b21aee0.gif)

Since only accessibility issues found in the preview window will reach our customers, I am going to close this issue. The area of the app we are discussing is not in the preview window and will only be used for internal testing.

Please let me know if you have any questions - thanks!

**Comment by vaishali1397** [comment]
Issue has been fixed in below test environment, hence adding A11yTTValidated tag.
**Test Environment:**
OS Version: 22H2 (Build 25330.1000)
**Application**: Adaptive Card Visualizer
https://user-images.githubusercontent.com/104483005/235426574-ed124170-4695-4c01-b34b-b9e7468d2fe1.mp4
